javascript/test_bolt12.js

```diff
@@ -0,0 +1,6 @@
+const {decode}=require('./index.js');
+file=require('../test-vectors/offers.json');
+for(let i=0;i<file.length;i++){
+    console.log(decode(file[i]['string']))
+    console.log("\n=======================================================================================================================\n")
```

Comment:
This tests that they all decode, now you need to test that the contents match...